vue/sbc-common-components/src/components/BaseAddress.vue

```diff
@@ -291,6 +261,15 @@ export default class BaseAddress extends Mixins(ValidationMixin, CountriesProvin
     return Boolean(this.schemaLocal && this.schemaLocal[prop] && this.schemaLocal[prop].isBC)
   }
 
+  /**
```

Comment:
i have no clue on the purpose..so a shot in dark... https://vuejs.org/v2/guide/forms.html#trim does it help in anyway?

Reply:
Good question. It sometimes works, sometimes for leading spaces and less often for trailing spaces and never for spaces between words. But it doesn's always work, while using a rule and a message always works and additionally shows the user the problem in case they edit an existing address with too much whitespace.